client/src/pages/ProfilePage/ProfilePage.js

```diff
@@ -20,23 +22,167 @@ export default function ProfilePage() {
   const [email, setEmail] = React.useState(null);
   const [profilePicture, setProfilePicture] = React.useState(null);
 
-  function handleDelete(tagIndex) {
-    const newTags = tags.filter((tag, i) => i !== tagIndex);
-    setTags(newTags);
+  const [dropDownOptions, setDropDownOptions] = React.useState([]);
+
+  function setProfileVariables(coop) {
+    setCoop(coop);
+    setLocation(coop['addr']);
+    setPhone(coop['phone_number']);
+    setTags(coop['tags']);
+    setMission(coop['mission_statement']);
+    setDescription(coop['description_text']);
+    setFbLink(coop['fb_link']);
+    setInstaLink(coop['insta_link']);
+    setEmail(coop['email']);
+    setWebsite(coop['website']);
+    setProfilePicture(coop['profile_pic']);
+    setName(coop['coop_name']);
   }
 
   React.useEffect(() => {
     async function fetchData() {
       try {
         const res = await axios.get('/api/coop');
         setProfileVariables(res.data);
+
+        function setDefaultTags(coop) {
```

Comment:
nit: can we move the logic in this function to `setProfileVariables` for consistency?